lib/katello/tasks/reimport.rake

```diff
@@ -16,6 +16,22 @@ namespace :katello do
     User.current = User.anonymous_admin #set a user for orchestration
     Dir.glob(Katello::Engine.root.to_s + '/app/models/katello/*.rb').each { |file| require file }
 
+    print "Fixing incorrect providers\n"
+    incorrect_provider_count = 0
+    error_msgs = []
```

Comment:
Originally I was just printing these rather than tallying them, but the task output is bulldozed by several screens of info. So I remember the data here and then output it below, when we're done.